### PR TITLE
feat: oav3 document store

### PR DESCRIPTION
**Pull request by yehjxraymond**

### CODE REVIEW

File: scripts/generate.v3.ts [hunk 1]
```diff
@@ -24,10 +27,11 @@ const ethereumDocumentConfig = {
     address: "0x1245e5B64D785b25057f7438F715f4aA5D965733",
     key: "0x416f14debf10172f04bef09f9b774480561ee3f05ee1a6f75df3c71ec0c60666",
   },
-  timeout: 60 * 1000, // 1 min timeout for contract to execute
+  timeout: 5 * 60 * 1000, // 5 min timeout for contract to execute
```
Comment (Nebulis) [inline scripts/generate.v3.ts:30]: 1min not enough ? xD

File: scripts/generate.v3.ts [hunk 2]
```diff
@@ -2,13 +2,16 @@
 import { __unsafe__use__it__at__your__own__risks__wrapDocument, v3 } from "@govtechsg/open-attestation";
```
Comment (Nebulis) [inline scripts/generate.v3.ts:2]: do we need to commit the generated files ?

File: src/verifiers/documentStatus/documentStore/ethereumDocumentStoreStatus.ts [hunk 3]
```diff
@@ -217,97 +218,147 @@ export const isRevokedOnDocumentStore = async ({
...
 };
-export const openAttestationEthereumDocumentStoreStatus: Verifier<
-  WrappedDocument<v2.OpenAttestationDocument> | WrappedDocument<v3.OpenAttestationDocument>
-> = {
-  skip: () => {
-    return Promise.resolve({
-      status: "SKIPPED",
-      type,
+
+const test: VerifierType["test"] = (document) => {
```
Comment (Nebulis) [inline src/verifiers/documentStatus/documentStore/ethereumDocumentStoreStatus.ts:235]: just to add that isWrappedV3Document is not correct as well (it doesn't do what you expect). We might need to discuss if we actually want to have such a helper in our lib :)

File: scripts/generate.v3.ts (hunk 2 above)
Comment (yehjxraymond) [reply scripts/generate.v3.ts:2]: yes pls, it takes a while to generate all of them, its just to generate test fixtures infrequently...

File: scripts/generate.v3.ts (hunk 1 above)
Comment (yehjxraymond) [reply scripts/generate.v3.ts:30]: Its enough for now, but have seen cases where the network choked

File: src/verifiers/documentStatus/documentStore/ethereumDocumentStoreStatus.ts (hunk 3 above)
Comment (yehjxraymond) [reply src/verifiers/documentStatus/documentStore/ethereumDocumentStoreStatus.ts:235]: "yes"

**Review by Nebulis — APPROVED** [review]

**Pull request merged**